lib/components/artist/header.js

```diff
@@ -4,7 +4,6 @@
 import React from 'react'
 import Relay from 'react-relay'
 import { NativeModules, StyleSheet, View, Dimensions } from 'react-native'
-const { ARTemporaryAPIModule } = NativeModules
```

Comment:
It seems like jest isn't able to recognize what this is unless we use `NativeModules.ARTemporaryAPIModule`.